app-office/whid/patches/whid-2.0.2~beta.patchset

```diff
@@ -0,0 +1,29 @@
+From 785e01790319302487834b37aa624e0d368e5319 Mon Sep 17 00:00:00 2001
+From: Mazbrili <mazbrili@yahoo.com>
+Date: Mon, 19 Aug 2019 14:25:43 +0700
+Subject: [PATCH] haiku specific location setting
+
+---
+ src/main.cpp | 5 ++++-
+ 1 file changed, 4 insertions(+), 1 deletion(-)
+
+diff --git a/src/main.cpp b/src/main.cpp
+index 805fa22..10bee6c 100644
+--- a/src/main.cpp
++++ b/src/main.cpp
+@@ -13,8 +13,11 @@ using namespace std;
+ void initSettings() {
+ 
+     QSettings settings;
+-
++#ifdef Q_OS_HAIKU
++  auto data_path = QStandardPaths::writableLocation(QStandardPaths::HomeLocation)+"/config/settings/TheLastViking/Whid";
++#elif
+     auto data_path = QStandardPaths::writableLocation(QStandardPaths::AppDataLocation);
```

Comment:
Shouldn't this already put files in correct position? IIRC the qt5 port of Haiku already returns the correct path for these situations.

Reply:
ok. delete because no longer needed